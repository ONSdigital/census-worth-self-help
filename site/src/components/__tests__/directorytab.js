import React from "react"
import renderer from "react-test-renderer"

import DirectoryTab from "../directorytab"

describe("DirectoryTab", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<DirectoryTab title="directory title" link="location" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})