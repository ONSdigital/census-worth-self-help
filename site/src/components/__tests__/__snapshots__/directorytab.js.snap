// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DirectoryTab renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "1j5mh0b",
      "next": undefined,
      "styles": "
          
    width: 100%;
    padding: 10px;
    min-height: 40px;
  
          background: rgb(120, 16, 137);
          border-left: 6px solid rgb(120, 16, 137);
      ",
    }
  }
>
  <a
    css={
      Object {
        "map": undefined,
        "name": "1tl9lwk",
        "next": undefined,
        "styles": "
          
    text-decoration: none;
    color: white;
  
          
    margin: 0px 2px;
  
        ",
      }
    }
    href="location"
  >
    directory title
  </a>
</div>
`;
